src/Workspaces/Core/Portable/SymbolKey/SymbolKey.SymbolKeyWriter.cs

```diff
@@ -165,7 +165,29 @@ private void WriteSymbolKey(ISymbol symbol, bool first)
 
                 if (!shouldWriteOrdinal)
```

Comment:
I don't get this variable...isn't it writing an ordinal? Or is "ordinal" and the "id" separate things here?